conf_files/pocs.yaml

```diff
@@ -66,6 +66,22 @@ messaging:
     cmd_port: 6500
     msg_port: 6510
 
+########################## Observations ########################################
+# Observations consist of a sequence of images and are grouped together
```

Comment:
This sentence feels a bit awkward to me. How would this be:

An observation folder contains a contiguous sequence of images of a target/field recorded by a single camera, with no slewing of the mount during the sequence; there may be tracking adjustments during the observation.